exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/QueryWrapper.java

```diff
@@ -115,7 +116,8 @@ public QueryResult run(final WorkManager workManager, final WebUserConnection we
     }
 
     // Return the QueryResult.
-    return new QueryResult(queryId, webUserConnection.columns, webUserConnection.results);
+    //return new QueryResult(queryId, webUserConnection.columns, webUserConnection.results);
```

Comment:
no need in leaving this string as a comment

Reply:
Done